src/ensembl/src/content/app/browser/Browser.tsx

```diff
@@ -211,11 +214,17 @@ export const Browser: FunctionComponent<BrowserProps> = (
     }
   }, [props.browserActivated]);
 
+  const closeDrawer = () => {
+    props.changeDrawerView('');
+    props.toggleDrawer(false);
+  };
```

Comment:
Why does `closeDrawer` belong to `Browser` component? It is passed from Browser to BrowserBar and TrackPanel components. Given that both `BrowserBar` and `TrackPanel` are connected to redux, why can't they import `closeDrawer` action creator directly from browser actions?

Reply:
Created a `closeDrawer` action in 8611954